frontend/src/container/MetricsApplication/Tabs/Overview.tsx

```diff
@@ -64,12 +91,96 @@ function Application({ getWidgetQueryBuilder }: DashboardProps): JSX.Element {
 		[handleSetTimeStamp],
 	);
 
-	const { topOperations, serviceOverview, topLevelOperations } = useSelector<
-		AppState,
-		MetricReducer
-	>((state) => state.metrics);
+	const {
+		data: serviceOverview,
+		error: serviceOverviewError,
+		isError: serviceOverviewIsError,
+		isLoading: serviceOverviewIsLoading,
+	} = useQuery(
+		[
+			`getServiceOverview`,
+			servicename,
+			getStep({ start: minTime, end: maxTime, inputFormat: 'ns' }),
```

Comment:
minMax time are sufficient to have key no need for the function call here

Reply:
Okay